### ISSUE TITLE
VS2019 16.8.0-16.8.2 "Fade out unreachable code" is not working

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#49870]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity2.visualstudio.com/t/VS2019-1680-1682-Fade-out-unreachab/1280943)._

---
[severity:It bothers me. A fix would be nice] [regression] [worked-in:!6.7.*]
Before VS2019 16.8, code that was unreachable would be faded out via the option "Fade out unreachable code". Since 16.8, code doesn't fade anymore out when this option is enabled.

It's quite easy to reproduce.

```c#
using System;

namespace Fading
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Hello World!");

return;

Console.WriteLine("Should be faded out"); // This line should be fade out, since it's unreachable
        }
    }
}
```

---
#### Feedback Bot on 12/8/2020, 10:06 PM: 

<p>We have directed your feedback to the appropriate engineering team for further evaluation. The team will review the feedback and notify you about the next steps.</p>

---
### Original Solutions
(no solutions)

**Comment by nibsi** [comment]
Still broken in `16.9.3`. Additionally, the `IDE0035` rule doesn't warn about unreachable code either.

Any news on this issue?

**Comment by CyrusNajmabadi** [comment]
@mavasani did we break something here with fading.  the analyzer seems to be working, so i'm not sure what's going on here.

**Comment by CyrusNajmabadi** [comment]
Note: we report the issue like so:

```c#
                context.ReportDiagnostic(DiagnosticHelper.CreateWithLocationTags(
                    Descriptor,
                    firstStatementLocation,
                    ReportDiagnostic.Default,
                    additionalLocations: ImmutableArray<Location>.Empty,
                    additionalUnnecessaryLocations: additionalLocations));
```

**Comment by LeroyK** [comment]
This is still broken in VS2022 17.2.6.

**Comment by CyrusNajmabadi** [comment]
Shows properly faded for me: 

![image](https://user-images.githubusercontent.com/4564579/198366383-6916da67-4275-49db-bf5d-c3f743c211fd.png)

**Comment by kamgmanGit** [comment]
Still BROKEN for me in in :
Microsoft Visual Studio Community 2022 (64-bit) - Current
Version 17.14.7

<img width="707" height="127" alt="Image" src="https://github.com/user-attachments/assets/e12d3d10-7d02-4533-af2e-e79a0b232b1f" />
